README.md

```diff
@@ -129,7 +175,8 @@ SQL database. If you need an example, you can use `people` and `wishes` models.
 Your answer...
 
 ```
-Replace this with your answer
+Schema, defines what columns it has within a table. We represent a one-to-many relationship in a SQL database by using tables and columns.
+
 ```
```

Comment:
A schema is a blueprint for a table. It defines the name of not only the table itself but its columns (along with each columns data type and optional constraints). Each column corresponds to an attribute that belongs to an instance of that table.